distributions/README.md

```diff
@@ -9,9 +9,12 @@ We welcome contributors to add configuration for additional platforms.
 
 ## Local development
 
-For local development (only), secrets may be stored in gradle properties.
-Add any test secrets to your local `.gradle/dataportability.secrets.properties` file. A
-sample is provided in `distributions/common/dataportability.secrets.example.properties`.
+For local development (only), secrets may be stored in environment variables.
```

Comment:
Maybe a quick example. We also have to ensure this works for the windows case

Reply:
Done.